src/styles/page/reset-catalog.scss

```diff
@@ -0,0 +1,3 @@
+* {
+  margin: 0;
+}
```

Comment:
Do global reset, not local

Reply:
reset-catalog.scss its the global reset for catalog.